### PR TITLE
Update to CAMP v2022.03.2

### PR DESCRIPTION
**Pull request by adayton1**
This version of CAMP has the changes we need for camp::tuple.

### CODE REVIEW

**Review by samuelpmishLLNL — APPROVED** [review]

**Comment by white238** [comment]
When updating the docker images you need to update the name of it as well in the azure pipeline configuration file.  Step 3 here tells you what to do:

https://serac.readthedocs.io/en/latest/sphinx/dev_guide/new_docker_image.html#update-add-docker-image

File: scripts/spack/packages/camp/package.py [hunk 1]
```diff
@@ -3,7 +3,17 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+import glob
```
Comment (white238) [inline scripts/spack/packages/camp/package.py:6]: Just to make sure nothing in here was added by you, @adayton1 ? It is all from an updated spack package, right?

Comment (adayton1) [reply scripts/spack/packages/camp/package.py:6]: Yes, that is right.

**Comment by adayton1** [comment]
> When updating the docker images you need to update the name of it as well in the azure pipeline configuration file. Step 3 here tells you what to do:
> 
> https://serac.readthedocs.io/en/latest/sphinx/dev_guide/new_docker_image.html#update-add-docker-image

Done.

**Review by white238 — APPROVED** [review]

**Pull request merged**